### ISSUE TITLE
Cannot add a badge

### ISSUE THREAD

**Issue by cibinlik** [GiocoApp/gioco#72]
I'm following the Youtube video on Windows 8.1 machine with Rails 4.1.8, Ruby 2.0.0p598 and Gioco 1.1.1 and when I try to create first badge with this line:

> rake gioco:add_badge[noob,100,teacher]

I get this error:

> rake aborted!
NameError: undefined local variable or method `teacher' for main:Object
C:/Sites/test_app/lib/tasks/gioco.rake:7:in `eval'
C:/Sites/test_app/lib/tasks/gioco.rake:7:in `eval'
C:/Sites/test_app/lib/tasks/gioco.rake:7:in `block (2 levels) in <top (required)
Tasks: TOP => gioco:add_badge
(See full trace by running task with --trace)

**Comment by joaomdmoura** [comment]
Weird. It's supposed to work. I'll check it @cibinlik  
Btw if you want, you can also try our pro version (http://gioco.pro). It has a free plan and a lot of cool features.

**Comment by cibinlik** [comment]
I'm a PM so cannot skip this:

Your Get Started with Gioco guide has codes starting with $ and if I click
to select the line, $ is also selected. I'm a Windows user and don't want
that!

And, I'm trying the pro one. Let me know if sth changes with the GitHub one.

Thanks ;)

**Comment by cibinlik** [comment]
Another tip;

On second step*To see full Gem Documentation *link should open in new tab,
or I lost the order.

**Comment by cibinlik** [comment]
Hey again,

https://github.com/GiocoApp/giocopro this page has no information about how
I can add badges/points to my users? or am I blind? :S

Merit and Gioco should improve their documentation to get the lead.
Otherwise people will just implement their own...

**Comment by joaomdmoura** [comment]
thank you for the feedback @cibinlik 
I'll remove the ```$``` and make the full documentation link to open in another tab.
@cibinlik Gioco is an event driven platform, you send the events to it, and create the badges, levels or points at the platform dashboard, it will handle everything else. After you finish the get started process you will have access to the dashboard.

**Comment by cibinlik** [comment]
With giocopro, am I gonna set the rules all from the dashboard?

**Comment by cibinlik** [comment]
I've implemented my own point/badge system but still interested in gioco. However, I would need a better documentation to keep moving. (I'm a Jr. Developer, Sr. Product Manager) ^^

**Comment by joaomdmoura** [comment]
@cibinlik Yes. Once that you integrated it with you application and you are sending the events (like a mixpanel), you can control and update everything using the dashboard with no need for programming any more

**Comment by joaomdmoura** [comment]
We are working hard on documentation. The 'get started' page is pretty straightforward, there are still some gaps. Thank you for your feedback, i you need some help just let us know! It will be easier than Github if you reach us at support@gioco.pro / joaomdmoura@gioco.pro
